packages/device-connector/tsconfig.json

```
@@ -14,6 +14,10 @@
     "declarationMap": true,
     "isolatedModules": true,
     "skipLibCheck": true,
-    "types": ["node"]
-  }
+    "types": ["node"],
+    "emitDeclarationOnly": false
```

Do we ever need declarations of the connector in any other packages? 
I think 

```suggestion
        "noEmit": false
```

should be fine.